scaffold/github/workflows/PantheonReviewAppsManual.yml

```
@@ -0,0 +1,72 @@
+name: "Pantheon Review Apps"
+
+on:
+  pull_request:
+    types: [ labeled, unlabeled, opened, synchronize, reopened]
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.head_ref || github.run_id }}
+  cancel-in-progress: true
+
+# You will also need to add the secrets used below to Dependabot secrets
+# if you want this to successfully run in auto-created PRs.
+permissions:
+  deployments: write
+  contents: read
+  pull-requests: read
+
+jobs:
+  Drainpipe-Deploy-Pantheon-Multidev:
+    if: contains(github.event.pull_request.labels.*.name, 'pantheon-multidev')
```

```suggestion
    if: (contains(github.event.pull_request.labels.*.name, 'pantheon-multidev') && ${{ vars.PANTHEON_MULTIDEV_MANUAL }} == "true") || ${{ vars.PANTHEON_MULTIDEV_MANUAL }} != "true"
```

Could we do something like that instead of duplicating the files? (haven't tested it so not sure if it'll work!)

---

Moved to https://github.com/Lullabot/drainpipe/pull/670#discussion_r1734704482